### PR TITLE
feat(atlas-web-components): add more types and implementing sizes

### PR DESCRIPTION
**Pull request by demogar**
# Feature

## Feature Description

- Add tertiary button
- Add destructive button
- Add :disabled property
- Add new examples

## Screenshot

image

## Checklist

- [ ] Tests
- [ ] Validated Against Design

### CODE REVIEW

File: .changeset/curvy-trains-whisper.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (nullcitizen) [inline .changeset/curvy-trains-whisper.md:1]: I'd love to know more about this file

Comment (demogar) [reply .changeset/curvy-trains-whisper.md:1]: these are automatically generated by [changesets](https://github.com/changesets/changesets)

- https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md#i-am-in-a-multi-package-repository-a-mono-repo
- https://github.com/changesets/changesets/blob/main/docs/detailed-explanation.md

File: apps/docs/examples/BasicButton/ExampleBasicButtonWithTrailingIcon copy.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { BasicButton } from "@productplan/atlas";
```
Comment (nullcitizen) [inline apps/docs/examples/BasicButton/ExampleBasicButtonWithTrailingIcon copy.js]: ope, is this file extra?

**Review by nullcitizen — APPROVED** [review]
hell yea

**Pull request merged**